tests/pytests/test_newhorizons_drivers.py

```diff
@@ -0,0 +1,259 @@
+    "Quaternions": [
+      [-0.040223695171375,
+       0.062038072129285,
+       0.97695635330468,
+       0.20022391388359]
+    ],
+    "AngularVelocity": [
+      [1.39581813997533e-05,
+       -1.15948738928358e-05,
+       2.21456402487273e-05]
+    ],
+    "ConstantFrames": [-98301, -98300, -98000],
+    "ConstantRotation": 
+    [-0.005452680629036, 
+     -0.99996036726125,
+     0.007037910250677, 
+     0.002999533810427,
+     -0.0070543385533461,
+     -0.99997061912063,
+     0.99998063534794,
+     -0.0054314099747325,
+     0.0030378799858676]
+  },
+  "BodyRotation": {
+    "TimeDependentFrames": [10023, 1],
+    "CkTableStartTime": 225940527.51631695,
+    "CkTableEndTime": 225940527.51631695,
+    "CkTableOriginalSize": 1,
+    "EphemerisTimes": [225940527.51631695],
+    "Quaternions": [
+      [-0.57817095054415,
+       0.13699803004374,
+       0.1729987472005,
+       -0.78550704973162]
+    ],
+    "AngularVelocity": [
+      [-6.24011073378906e-07,
+       -1.76837707421929e-05,
+       3.7102455825449e-05]
+    ]
+  },
+  "InstrumentPosition": {
+    "SpkTableStartTime": 225940527.51631695,
+    "SpkTableEndTime": 225940527.51631695,
+    "SpkTableOriginalSize": 1,
+    "EphemerisTimes": [225940527.51631695],
+    "Positions": [
+      [-2390849.002579,321807.33761069,-141054.73789822]
+    ],
+    "Velocities": [
+      [6.3943726561674,-22.16679208555,-8.6097977343012]
+    ]
+  },
+  "SunPosition": {
+    "SpkTableStartTime": 225940527.51631695,
+    "SpkTableEndTime": 225940527.51631695,
+    "SpkTableOriginalSize": 1,
+    "EphemerisTimes": [225940527.51631695],
+    "Positions": [
+      [311036228.79889,679984041.50957,283865335.82619
+      ]
+    ],
+    "Velocities": [
+      [
+      4.7547216698047,-0.80026582585286,-0.11480667580455
+      ]
+    ]
+  }
+}
+
+@pytest.fixture()
+def test_kernels(scope="module"):
+    kernels = get_image_kernels("lor_0034974380_0x630_sci_1")
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    yield updated_kernels
+    for kern in binary_kernels:
+        os.remove(kern)
+
+def test_newhorizons_load(test_kernels, isis_compare_dict):
+    class AleJsonEncoder(json.JSONEncoder):
+        def default(self, obj):
+            if isinstance(obj, set):
+                return list(obj)
+            if isinstance(obj, np.integer):
+                return int(obj)
+            elif isinstance(obj, np.floating):
+                return float(obj)
+            elif isinstance(obj, np.ndarray):
+                return obj.tolist()
+            elif isinstance(obj, datetime.date):
+                return obj.isoformat()
+            return json.JSONEncoder.default(self, obj)
+
+    label_file = get_image_label("lor_0034974380_0x630_sci_1", "isis")
+    with NewHorizonsLorriIsisLabelNaifSpiceDriver(label_file, props={'kernels': test_kernels}) as driver: 
+        isd_str = json.dumps(to_isis(driver), cls=AleJsonEncoder)
+    isis_obj = json.loads(isd_str)
```

Comment:
You should use ale load or loads here instead of instantiating the driver and calling the formatter.